src/components/StaffList/index.jsx

```diff
@@ -33,17 +95,108 @@ class StaffList extends React.Component {
     }, () => this.props.input.onChange(this.state.staffList));
   }
 
+  /**
+   * The callback method for when a user enters a value in the autocomplete list.
+   * Responsible for updating the redux state of selected options.
+   *
+   * @param event - unused but a required parameter for autosuggest component
+   * @param suggestion - the entered suggestion
+   */
+  onSuggestionEntered(event, { suggestion }) {
+    const { staffList } = this.state;
+    const newInstructor = (suggestion.id === 'new');
+    const instructorAlreadySelected =
+      staffList.some(staff => staff.uuid === suggestion.text.uuid);
+    if (instructorAlreadySelected) {
+      // do we want to message the user?
```

Comment:
I think we should get a decision from carol or @staubina on this, or just delete this if for now